app/presenters/nav_tab/global_link_collection.rb

```diff
@@ -0,0 +1,43 @@
+class NavTab::GlobalLinkCollection
+
+  include Rails.application.routes.url_helpers
+
+  attr_reader :ability
+
+  delegate :can?, to: :ability
+
+  def initialize(ability)
+    @ability = ability
+  end
+
+  def links
+    [
+      admin_cross_facility_users,
+      global_settings,
+      admin_cross_facility_billing,
+    ].select(&:present?)
+  end
+
+  private
+
+  def admin_cross_facility_billing
+    if can?(:manage_billing, Facility.cross_facility)
+      global_tab(:admin_billing, facility_transactions_path("all"))
+    end
+  end
+
+  def admin_cross_facility_users
+    if can?(:manage_accounts, Facility.cross_facility) && can?(:manage_users, Facility.cross_facility)
```

Comment:
Currently I don't think so. Account Managers have both abilities, but that's all I see. We grant `:manage` to `Facility` if a user is a `manager_of?` a facility, but I don't expect anyone to be a manager of `Facility.cross_facility`.

That said, this could be `||` rather than `&&`.